samples/dlp_inspect_file.js

```diff
@@ -0,0 +1,91 @@
+/**
```

Comment:
Is there a sample in another file that should be removed? Or is this really additive?

Reply:
There are samples in other files that should be removed.

Reply:
Sweet.  Should we include that in this PR?

Reply:
yeah, I'm not done, some people wanted to see this so I threw what I had so far into a PR

Reply:
(This probably won't be merged for a while, if at all)

Reply:
Done